sqlmesh/core/engine_adapter/spark.py

```diff
@@ -334,39 +329,41 @@ def _get_data_objects(
             for row in results  # type: ignore
         ]
 
+    @property
+    def _spark_semver(self) -> t.Tuple[int, int, int]:
```

Comment:
[Nit] Doesn't look like we care about the patch version anywhere.

Reply:
Removed